### PR TITLE
cb-10-yelp Call the yelp api for things to do

### PR DESCRIPTION
**Pull request by jadendick**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
New feature

* **What is the current behavior?What does the code do before you made the new changes?** 
The only route was '/' and it just returned "Hello world"

* **What is the new behavior?**
I added a '/yelp' route that returns a yelp search of things to do in a given city

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No

* **Other information**:

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -10,6 +10,22 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/yelp',(req,res)=>{
```
Comment (Codubee) [inline backend/index.js]: Can we name the endpoint something a little bit more descriptive. If I knew nothing about this api other than the name of it, would '/yelp' tell me what this api is doing?

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Jaden! API works as expected. just one comment.

File: backend/index.js (hunk 1 above)
Comment (jadendick) [reply backend/index.js]: I changed it to /yelpGetThingsToDo, is that too long or anything. Is there a certain format I should follow?

**Review by Codubee — APPROVED** [review]
LGTM

**Pull request merged**